kuebiko/pipeline/nspl/flow.py

```diff
@@ -0,0 +1,215 @@
+from typing import Dict, Literal
```

Comment:
When I try to run this flow I get the following error:

`selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH.`

Reply:
Hi @Juan-Mateos, I also had an issue with chromedriver before being able to run  `kuebiko/pipeline/nspl/flow.py`. Did you try adding chromedriver to PATH?

Reply:
I had the same issue @Juan-Mateos and fixed it as @Jack-0-0 has described.
Perhaps worth noting somewhere in the comments or documentation.

Also (slightly off topic): @bishax What would be the best practice for using chromedriver, if we'd like to run a similar flow on the cloud? Seems like setting up a chromedriver executable on an ec2 would be a nuisance. Perhaps using something like `webdriver-manager` ?